### DISCUSSION TITLE
Top right icons?

### DISCUSSION THREAD

**Discussion by dasche** [rumboalla/apkupdater#498] [Q&A]
I request the developers to add some text to the top right icons on home page of app. It's totally confusing. Can't understand what those icons mean. Thanks 
![Screenshot_2024-01-17-21-51-21-91_3bb3bffa89a9f6c4fe870b0aa2a0ce96](https://github.com/rumboalla/apkupdater/assets/1790918/731a73d3-ef85-4808-b6df-7f8c30df8619)

**Comment by rumboalla** [comment]
Longpress

**Reply by dasche** [reply]
Thanks. But how could someone know to long press? Please Try adding button labels
